src/GCHP_GridComp/GCHPctmEnv_GridComp/GCHPctmEnv_GridCompMod.F90

```diff
@@ -676,8 +694,11 @@ subroutine Run ( GC, IMPORT, EXPORT, CLOCK, RC )
       ! Use dry pressure at the start of the timestep to calculate mass
       ! fluxes. GMAO method uses mid-step UC, VC and PLE?
       PLEr8 = 1.00d0*(DryPLE0r8)
+      if (.not. firstRun) THEN
       call fv_computeMassFluxes(UCr8, VCr8, PLEr8, &
                                    MFXr8, MFYr8, CXr8, CYr8, dt)
+      endif
+      firstRun = .false.
```

Comment:
This change here is causing differences in the GCHP benchmark run. Could you explain why you put this here? Is it only for adjoint?

Reply:
This has to do with the reversal of the high-level operators; I have put it in an #ifdef ADJOINT block.